vhdbuilder/packer/install-dependencies.sh

```diff
@@ -300,8 +300,10 @@ unpackAzureCNI() {
 
 #must be both amd64/arm64 images
 VNET_CNI_VERSIONS="
-1.5.11
 1.4.43.1
```

Comment:
1.4.43.1  and 1.5.11 are what have the (not real) cve in them right? So after our toggle release we can clear these then.

Reply:
correct